lib/ts/recipe/dashboard/api/listTenants.ts

```diff
@@ -43,6 +48,12 @@ export default async function listTenants(
 ): Promise<Response> {
     let tenantsRes = await Multitenancy.listAllTenants(userContext);
     let finalTenants: TenantListTenantType[] = [];
```

Comment:
the public tenant will have all three login methods as enabled by default.. which means in the UI, you will see email password and passwordless methods to create users, even if the user has not init those recipes!

Reply:
done